catalyst/tests/test_finetune2.py

```diff
@@ -31,7 +31,9 @@ def get_loggers(self):
             "tensorboard": dl.TensorboardLogger(logdir=self._logdir),
         }
         if SETTINGS.ml_required:
```

Comment:
```suggestion
        if SETTINGS.mlflow_required:
```
looks like here is a typo